lib/ci/ci_type_parser.js

```diff
@@ -77,8 +77,8 @@ const CI_TYPES = new Map([
   }],
   [LITE_PR, {
     name: 'Lite PR',
-    jobName: 'node-test-pull-request-lite',
-    pattern: /job\/node-test-pull-request-lite\/(\d+)/,
+    jobName: 'node-test-pull-request-lite-pipeline',
```

Comment:
I believe the pattern is actually `\/node-test-pull-request-lite-pipeline\/(\d+)\/pipeline`?

Reply:
Yep, the pattern was indeed as you suggested updated and added it as new item.